lib/iris/tests/integration/test_pp.py

```diff
@@ -587,5 +588,56 @@ def test_as_pairs(self):
             self.assertEqual(cube.attributes['pseudo level'], 3)
 
 
+class TestSaveLBPROC(tests.IrisTest):
+
+    def create_cube(self):
+        cube = Cube(np.zeros((2, 3, 4)))
+        cube.add_dim_coord(
+            DimCoord(standard_name='time',
+                     units=Unit('days since 0001-01-01 00:00:00',
+                                calendar='gregorian'),
+                     points=np.arange(2),),
+            0)
+        cube.add_dim_coord(DimCoord(standard_name='longitude',
+                                    units='degrees',
+                                    points=np.arange(3),), 1)
```

Comment:
@ajdawson I recognise that var_name and standard_name are different: but for the purposes of these tests, they're functionally the same.  If there's no name at all on the coordinate, the relevant LBPROC isn't set.  However, var_name is sufficient to trigger setting LBPROC, which is all we're interested in here (the test failure below is another timed out regrid).

Now, if you're concerned that future changes around this area of the code (as I understand it, the pp_save_rules are being migrated to python?) will break the usage of var_name here, that's fine: let me know, and I'll revert to standard_name.